things/api.py

```diff
@@ -468,7 +468,7 @@ def today(**kwargs):
     See `things.api.tasks` for details on the optional parameters.
     """
     database = pop_database(kwargs)
-    if not database.was_modified_today():
+    if not database.was_modified_today():  # pragma: no cover
```

Comment:
@AlexanderWillner Might use monkeypatching here?

Reply:
explain? ;)

Reply:
https://docs.python.org/3/library/unittest.mock.html

Reply:
https://en.wikipedia.org/wiki/Monkey_patch

Reply:
Ok, I know mocking, but didn't hear the term monkey patching before. Created a ticket for this for now.

Reply:
👍